src/mono/netcore/Makefile

```diff
@@ -1,38 +1,55 @@
-DOTNET=../../../.dotnet/dotnet
+DOTNET := $(shell bash init-tools.sh | tail -1)
```

Comment:
So now all commands will trigger dotnet install if `.dotnet` is missing or was removed (because all commands need it to run targets)